### DISCUSSION TITLE
Regional control?

### DISCUSSION THREAD

**Discussion by Nott89** [lllyasviel/stable-diffusion-webui-forge#1867] [Q&A]
I'm new to Forge, migrated from Comfyui, I know that generally imaging with multiple subjects tied to specific Lora can be tricky without the right addons or extensions, but I can't find one that works with the current version of Forge? Can anyone give me some advice?

**Comment by Haoming02** [comment]
~~**Shameless Self Promotion:**~~ https://github.com/Haoming02/sd-forge-couple

**Reply by Nott89** [reply]
Thanks, I didn't find it in the extensions list and had to install it manually, but it seems to work very well. It has a really nice UX :D

**Reply by HMRMike** [reply]
It's brilliant and the operation is very comfortable, the way the masks are separated and can be reloaded and edited on the fly, with the prompt automatically assigned to each correctly, is awesome.

**Comment by SillySilk** [comment]
Thank you!!!!!
